### DISCUSSION TITLE
VRoidSDK モデルロード時のオプションについて

### DISCUSSION THREAD

**Discussion by goshawk329** [pixiv/vroid-sdk-developers#149] [Q&A]
いつも大変お世話になっております。

VRoidSDKでロードするモデルについて、ModelLoaderにControlRigGenerationOptionを追加して利用させて頂きたいのですが、これはSDKガイドライン違反にあたりますでしょうか。また、今後のSDKリリースで上記オプションに対応する予定はございますか。

よろしくお願いいたします。

**Comment by ytomi-pixiv** [comment]
ご連絡ありがとうございます。

> ModelLoaderにControlRigGenerationOptionを追加して利用させて頂きたいのですが、これはSDKガイドライン違反にあたりますでしょうか

ModelLoader に ControlRigGenerationOption を追加することは、現状の SDK ガイドラインには違反しておりません。

> また、今後のSDKリリースで上記オプションに対応する予定はございますか。

正規化されていないモデルへの対応について、今回のフィードバックを参考に課題として再認識いたしました。具体的な対応方針については現在模索中であり、現状ではお返事できる段階になく、ご理解いただけますと幸いです。

**Comment by goshawk329** [comment]
ご返信ありがとうございます。
承知いたしました。
